### PR TITLE
fix: dependabot alert for yaml

### PR DESCRIPTION
**Pull request by 0618**
#### Description of changes

fix https://github.com/aws-amplify/maplibre-gl-js-amplify/security/dependabot/18

#### Checklist

- [x] PR description included
- [ ] `yarn test` passes
- [ ] Tests are [changed or added]
- [ ] Relevant documentation is changed or added (and PR referenced)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -107,6 +107,7 @@
     "maplibre-gl-draw-circle": "^0.1.1"
   },
   "resolutions": {
-    "json5": "^2.2.2"
+    "json5": "^2.2.2",
+    "yaml": "2.2.2"
   }
-}
+}
```
Comment (reesscot) [inline package.json:113]: missing newline
```suggestion
}

```

**Review by reesscot — DISMISSED** [review]

File: yarn.lock [hunk 2]
```diff
@@ -8955,10 +8955,10 @@ yallist@^4.0.0:
   resolved "https://registry.npmjs.org/yallist/-/yallist-4.0.0.tgz"
   integrity sha512-3wdGidZyq5PB084XLES5TpOSRA3wjXAlIWMhum2kRcv/41Sn2emQ0dycQW4uZXLejwKvg6EsvbdlVL+FYEct7A==
 
-yaml@^1.10.0:
-  version "1.10.2"
-  resolved "https://registry.npmjs.org/yaml/-/yaml-1.10.2.tgz"
-  integrity sha512-r3vXyErRCYJ7wg28yvBY5VSoAF8ZvlcW9/BwUzEtUsjvX/DKs24dIkuwjtuprwJJHsbyUbLApepYTR1BN4uHrg==
+yaml@2.2.2, yaml@^1.10.0:
+  version "2.2.2"
+  resolved "https://registry.yarnpkg.com/yaml/-/yaml-2.2.2.tgz#ec551ef37326e6d42872dad1970300f8eb83a073"
```
Comment (sreeramsama) [inline yarn.lock:8960]: Not sure why registry changed but you can try setting registry locally to https://registry.npmjs.org/ and update yarn.lock file again.

Comment (reesscot) [reply yarn.lock:8960]: +1

Comment (0618) [reply yarn.lock:8960]: Good catch! That's a question I had for this package. The package has a `yarn.lock` so I used `yarn build`, but in the `package.json`, all scripts use `npm` 🤷

Comment (reesscot) [reply yarn.lock:8960]: Maybe we should standardize on yarn?

Comment (sreeramsama) [reply yarn.lock:8960]: Yeah.
For now, can you try
```
yarn config set registry https://registry.npmjs.org
yarn
```

Comment (sreeramsama) [reply yarn.lock:8960]: Looks like we use a mix of both - npm for builds, and yarn for publish. Not sure why.

Comment (0618) [reply yarn.lock]: NVM, I just changed the registery to `npm` to make the PR simple

**Review by sreeramsama — APPROVED** [review]

**Review by reesscot — APPROVED** [review]

**Pull request merged**